modules/xmpp/moderator.js

```diff
@@ -210,11 +210,12 @@ Moderator.prototype.createConferenceIq = function() {
                 value: this.options.conference.minBitrate
             }).up();
     }
-    if (this.options.conference.openSctp !== undefined) {
+    if (this.options.conference.openBridgeChannel !== undefined) {
```

Comment:
Or false. If the user prefers a websocket, or nothing at all, we should pass false here.